src/utils/videoTracking.ts

```diff
@@ -92,6 +94,8 @@ function addRunTimeParams(trackingParams: TrackData): TrackData {
 	trackDataObject[PROPERTY_NAMES.VIDEO_DURATION] = getDuration();
 	trackDataObject[PROPERTY_NAMES.VIDEO_PLAYER] = getPlayerId();
 	trackDataObject[PROPERTY_NAMES.VIDEO_VIEW_STATE] = getJWViewState();
+	trackDataObject[PROPERTY_NAMES.VIDEO_IS_EMBEDDED] = getIsEmbed();
+	trackDataObject[PROPERTY_NAMES.VIDEO_QUALITY_MANIFEST] = getJWQualityManifest();
```

Comment:
So are we storing the available quality levels for the player, whenever it loads?

Reply:
This is the _possible_ qualities that the video can player. It's available as soon as the player loads. Which all of these events fire after

Reply:
Just saw it in the doc - https://docs.google.com/spreadsheets/d/1zBUIt8fxmjEItf9HiW9TU1EZnopSLju_TO-BEEM-WuE/edit#gid=1564524057

If I understand it correctly, we'll be sending these the player quality values on each tracking event. The thing I'm uncertain about is whether we'll also send the current quality that the player is watching the video at, not just all the available ones.

Reply:
Yup, already done via `video_current_quality`